search/serializers_bulk_test.py

```diff
@@ -0,0 +1,333 @@
+"""Tests for elasticsearch bulk serializers"""
```

Comment:
Most of these are pre-existing tests moved over to a new file because the original was getting too long (>1000 lines)